book/typst/common/settings-slides-white.typ

```diff
@@ -0,0 +1,118 @@
+#import "./defaults.typ": env_lookup_all, restrict, config
+#import "../libs/_workbook.typ": setup
+// Slides are similar to a workbook, so we use the workbook functions as a basis for the slides.
+#let workbook = setup(..config, banner_color: color.rgb("#00a2cb"), def_color: color.rgb("#8dc73e"))
+
+#let active_env = "slides"
+
+// Bind `lookup_def` with the current definition environment so they appear boxed correctly.
+#import "./definitions.typ": show_def
+/// Show the definition with the given name. `name` is a string.
+#let show_def = show_def.with(definition_env: workbook.definition)
+
+#let slidetitlebox = color.rgb("#00647d")
+#let slideshade = color.rgb("#ffffff")
+#let (sans, serif, question_counter) = workbook
+
+
+/// Render a slide panel with the given content.
+/// - `content` (content): The content to render on the slide.
+/// - `autosize` (bool): Whether to attempt to automatically size the content to fit on a single slide.
+#let slide(content, autosize: true, force_two_column: false, force_scale: none) = {
```

Comment:
It'd be nice if there were a way to import this function instead. That way we don't have to update the algorithm in multiple places.

Maybe there can be a `settings-slides-common.typ`?

Reply:
How about we pass a `bottom_margin_color` in?